js/ui/multi_view.js

```diff
@@ -319,16 +320,42 @@ const MultiView = CollectionWidget.inherit({
         });
     },
 
+    _findBoundaryAvailableIndices() {
+        const items = this.option('items');
+
+        let firstIndex;
+        let lastIndex;
+
+        items.forEach((item, index) => {
+            const isDisabled = Boolean(item?.disabled);
```

Comment:
It is necessary, because in some tests we use null-items, for example: [[tabPanel.markup.tests](https://github.com/DevExpress/DevExtreme/blob/23_1/testing/tests/DevExpress.ui.widgets/tabPanel.markup.tests.js#L153)].